push-subscription-management/server.js

```diff
@@ -0,0 +1,61 @@
+var webPush = require('web-push');
+// collection of endpoints stored in memory (in production it should be
+// a database)
+var subscriptions = [];
```

Comment:
this is more a hashtag than a user - just so a client is identified in the server